api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/ClaimServiceTest.java

```diff
@@ -462,6 +513,14 @@ void shouldSaveClaimantWhenEligibilityThrowsException() {
         assertThat(eventCaptor.getValue().getEventMetadata().get(FailureEvent.FAILED_EVENT_KEY)).isEqualTo(ClaimEventType.NEW_CLAIM);
     }
 
+    private ClaimRequest aClaimRequestForClaimant(Claimant claimant) {
```

Comment:
Ah, this should be in a Test Data Factory so you can reuse above as per my other comment.

Reply:
done